### DISCUSSION TITLE
config.ini

### DISCUSSION THREAD

**Discussion by andrewcrosse** [Louisvdw/dbus-serialbattery#633] [General]
hi all
i can alter nano /data/etc/dbus-serialbattery/utils.py 
but when i type in 
nano /data/etc/dbus-serialbattery/config.ini  nothing comes up?
running latest version

**Comment by andrewcrosse** [comment]
editing using putty

**Comment by rperpe** [comment]
Which version are you using exactly? 0.14.3?

https://louisvdw.github.io/dbus-serialbattery/general/install#settings-locationpath

> Copy the values from /data/etc/dbus-serialbattery/config.default.ini to /data/etc/dbus-serialbattery/config.ini you want to change. All options can also be copied from [here](https://github.com/Louisvdw/dbus-serialbattery/blob/jkbms_ble/etc/dbus-serialbattery/config.default.ini).

(config.ini must contain [DEFAULT] in the first line as far as I know)

**Reply by andrewcrosse** [reply]
thanks yes 0.14.3
ill have another play around see where i get

**Comment by kompenhans** [comment]
hi all

"Driver version >= v1.0.0
Copy the values from /data/etc/dbus-serialbattery/config.default.ini to /data/etc/dbus-serialbattery/config.ini you want to change. All options can also be copied from [here](https://github.com/Louisvdw/dbus-serialbattery/blob/jkbms_ble/etc/dbus-serialbattery/config.default.ini)."

=> on a fresh install of latest release there is no config.default.ini 
=> the ULR https://github.com/Louisvdw/dbus-serialbattery/blob/jkbms_ble/etc/dbus-serialbattery/config.default.ini (location to copy a template) is also invalid.

best regards
// Dirk

**Reply by mr-manuel** [reply]
The latest release is [`v.0.14.3`](https://github.com/Louisvdw/dbus-serialbattery/releases/tag/v0.14.3). The latest pre release is [`v1.0.20230508beta`](https://github.com/Louisvdw/dbus-serialbattery/releases/tag/v1.0.20230508beta).

The broken link was reported with https://github.com/Louisvdw/dbus-serialbattery/issues/634 and is fixed now.

**Comment by Kris-9928** [comment]
After you make changes in config.ini and reboot the device (or run /data/etc/dbus-serialbattery/reinstall-local.sh) where these changes are written to? Thanks

**Reply by mr-manuel** [reply]
With v1.x.x

https://github.com/Louisvdw/dbus-serialbattery/blob/fd0fc1a193427df6ca0fadd141aa57826f761819/etc/dbus-serialbattery/reinstall-local.sh#L104
